src/components/PlatformNavigator/__tests__/PlatformNavigator.test.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { PlatformNavigator } from '../index';
+import userEvent from '@testing-library/user-event';
+
+const routerMock = {
+  __esModule: true,
+  useRouter: () => {
+    return {
+      pathname: ''
+    };
+  }
+};
+
+jest.mock('next/router', () => routerMock);
+
+describe('PlatformNavigator', () => {
+  const component = (
+    <PlatformNavigator currentPlatform={'react'} isPrev={true} />
+  );
+
+  it('should render the PlatformNavigator component', async () => {
+    render(component);
+
+    const navigator = await screen.findByText('Choose your framework/language');
+    expect(navigator).toBeInTheDocument();
+  });
+
+  it('should show the selected platform as React', async () => {
+    render(component);
+
+    const navigatorComponent = await screen.findByText(
+      'Choose your framework/language'
+    );
+    const platform =
+      navigatorComponent.parentElement?.getElementsByTagName('button')[0]
+        .textContent;
+
+    expect(platform).toBe('React');
+  });
+
+  it('should open dropdown on click', async () => {
+    render(component);
+
+    const platformButton = await screen.getByRole('button');
+    expect(platformButton.getAttribute('aria-expanded')).toBe('false');
+
+    userEvent.click(platformButton);
+    expect(platformButton.getAttribute('aria-expanded')).toBe('true');
+  });
+
+  it('should change platform on selection', async () => {
```

Comment:
In this test we're not actually changing the platform right? We're just tabbing through the menu to make sure the correct platform is focused?

Maybe we could have a more descriptive test name like
```suggestion
  it('should tab through platforms in dropdown', async () => {
```

Reply:
+1